src/all.test.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-env jest */
+/* global spyOn */
+import HMRCFrontend from './all';
+import * as characterCountModule from './components/character-count/character-count';
+
+describe('Init All', () => {
+  let testScope;
+  beforeEach(() => {
+    testScope = {
+      elemsToWipe: [],
+    };
+  });
+  afterEach(() => {
+    jest.resetAllMocks();
+    testScope.elemsToWipe.forEach((elem, key) => {
+      const parent = elem.parentNode;
+      if (parent) {
+        parent.removeChild(elem);
+      } else {
+        console.warn('failed to remove child', key);// eslint-disable-line no-console
```

Comment:
Is this likely to happen? Just wondering why something like this would be in a test?

Reply:
Fair, I hit that condition when I tried to remove the same item twice.  I think I might throw instead.